### ISSUE TITLE
Infinite difference in plan

### ISSUE THREAD

**Issue by bigwheel** [yumemi-inc/terraform-provider-slackapp#1]
### Situation

```terraform
resource "slackapp_application" "this" {
  manifest = file("../manifest.json")
}
```

manifest.json:
https://github.com/slack-samples/bolt-ts-starter-template/blob/57b023d4c33ce0e821a619d4581dc15514497e63/manifest.json

### What happens

After first apply, there is infinite diff in plan/apply.

```
Terraform will perform the following actions:

  # slackapp_application.this will be updated in-place
  ~ resource "slackapp_application" "this" {
        id                  = "A06058VCT8V"
      ~ manifest            = jsonencode(
          ~ {
              + _metadata           = {
                  + major_version = 1
                  + minor_version = 1
                }
              ~ settings            = {
                  + token_rotation_enabled = false
                    # (4 unchanged attributes hidden)
                }
                # (3 unchanged attributes hidden)
            }
        )
        # (2 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

### About `token_rotation_enabled`

This `settings.token_rotation_enabled` is not documented yet ( https://api.slack.com/reference/manifests#settings ).
But feature exists.
![2023-10-09_10-48](https://github.com/yumemi-inc/terraform-provider-slackapp/assets/826646/657eabe3-c491-479f-8785-6845d9f19618)
And also in manfiest.
![2023-10-09_10-50](https://github.com/yumemi-inc/terraform-provider-slackapp/assets/826646/c1ba71d6-fe19-405e-a700-204a69ff295a)

I guess we have to add `token_rotation_enabled` to [settings structs](https://github.com/search?q=repo%3Ayumemi-inc%2Fterraform-provider-slackapp%20settings&type=code).

### About `_metadata`

`_metadata` exists in [reference](https://api.slack.com/reference/manifests#metadata) but removed when manifest pushed.
Therefore, it is better to ignore _metadata in the difference check.

----

Thank you for great provider!

**Comment by siketyan** [comment]
Thank you for reporting this! And sorry for the late response (I wasn't watching this repo). Will investigate and try to fix the issues.

**Comment by siketyan** [comment]
@bigwheel
Just released v0.2.4 including the fix of this issue. 🎉

**Comment by bigwheel** [comment]
@siketyan Thank you for your work!
I will test v0.2.4 in next chance.
